pavelib/assets.py

```diff
@@ -718,6 +705,28 @@ def execute_compile_sass(args):
         )
 
 
+def get_parsed_option(options, name, default=None):
+    """
+    Options to a command are provided as string value. Convert it to list where a list is expected
+    after fetching an option specified by name
+    Args:
+        options: Command line arguments passed via paver command.
+        name: name of option to parse
+        default: if option not in options, return default
+    """
+    option = default
+    try:
+        option = getattr(options, name)
```

Comment:
yeah I got that, changes are in progress. Actually I wrote this before I was passing default in this method. After passing default there is no need of it.